test/LocalNeighborhood/test_ln.py

```diff
@@ -50,3 +46,17 @@ def test_format_error(self):
                                output_file=OUT_FILE)
 
     # Write tests for the Local Neighborhood run function here
+def test_localneighborhood_run():
+    """
+    Run Localneighborhood with the run function and check the output matches the expected output
+    """
+    # Remove the output file if it exists
+    out_path = Path(OUT_FILE)
+    out_path.unlink(missing_ok=True)
+    # Run the run function in LocalNeighborhood
+    local_neighborhood.run(
+        nodes=TEST_DIR + 'input/ln-nodes.txt',
+        network=TEST_DIR + 'input/ln-network.txt',
+        output_file=OUT_FILE
+    )
+    assert out_path.exists()
```

Comment:
This should also have a missing test to make sure that `LocalNeighborhood#run` is erroring on missing inputs as well.

(I also missed this in #291, but this should have an accompanying singularity test as well).